ui-framework-documentation.md

```diff
@@ -228,7 +304,7 @@ sub run {
 }    
 ``` 
 
-But in case if need to verify some negative cases the method cannot be
```

Comment:
I had hard time to make sense out of it, I think in ecuh case nowadays we would just create a different method in the controller (??)